erp42_control:
  # Joint 상태 퍼블리셔
  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50

  # ✅ 뒷바퀴 구동용 Velocity 컨트롤러
  left_back_wheel_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: left_back_wheel_joint
    pid: {p: 10.0, i: 0.01, d: 0.1}

  right_back_wheel_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: right_back_wheel_joint
    pid: {p: 10.0, i: 0.01, d: 0.1}

  # ✅ 앞바퀴 조향용 Position 컨트롤러
  left_steer_wheel_controller:
    type: "position_controllers/JointPositionController"
    joint: left_front_disk_wheel_joint
    pid: {p: 3.0, i: 0.01, d: 0.05}

  right_steer_wheel_controller:
    type: "position_controllers/JointPositionController"
    joint: right_front_disk_wheel_joint
    pid: {p: 3.0, i: 0.01, d: 0.05}
